qt/scientific_interfaces/Indirect/IndirectPlotOptionsView.h

```diff
@@ -84,6 +109,8 @@ private slots:
   std::unique_ptr<QStringListModel> m_suggestionsModel;
   std::unique_ptr<QCompleter> m_completer;
   std::unique_ptr<Ui::IndirectPlotOptions> m_plotOptions;
+
+  IIndirectPlotOptionsPresenter *m_presenter;
```

Comment:
Isn't it better if this raw pointer is initialized with nullptr at the constructor till a presenter is subscribed to it?